doc/dev/dev_cluster_deployement.rst

```diff
@@ -37,49 +65,69 @@ Options
 
     Specifies monitor *ip* address and *port*.
 
+.. option:: --memstore
+
+    Use memstore as the objectstore backend for osds
+
+.. option:: --multimds <count>
+
+    Allow multimds with maximum active count.
+
 .. option:: -n, --new
 
     Create a new cluster.
 
-.. option:: -o config
+.. option:: -N, --not-new
 
-    Add *config* to all sections in the ceph configuration.
+    Reuse existing cluster config (default).
 
 .. option:: --nodaemon
 
     Use ceph-run as wrapper for mon/osd/mds.
 
-.. option:: --smallmds
+.. option:: --nolockdep
 
-    Configure mds with small limit cache size.
+    Disable lockdep
 
-.. option:: -x
+.. option:: -o <config>
 
-    Enable Cephx (on by default).
+    Add *config* to all sections in the ceph configuration.
 
-.. option:: -X
+.. option:: --rgw_port <port>
 
-    Disable Cephx.
+    Specify ceph rgw http listen port.
 
-.. option:: -d, --debug
+.. option:: --rgw_frontend <frontend>
+
+    Specify the rgw frontend configuration (default is civetweb).
 
-    Launch in debug mode
+.. option:: --rgw_compression <compression_type>
+
+    Specify the rgw compression plugin (default is disabled).
+
+.. option:: --smallmds
+
+    Configure mds with small limit cache size.
+
+.. option:: --short
+
+    Short object names only; necessary for ext4 dev
 
 .. option:: --valgrind[_{osd,mds,mon}] 'valgrind_toolname [args...]'
 
     Launch the osd/mds/mon/all the ceph binaries using valgrind with the specified tool and arguments.
 
-.. option:: --bluestore
+.. option:: --without-dashboard
 
-    Use bluestore as the objectstore backend for osds
+    Do not run using mgr dashboard.
 
-.. option:: --memstore
+.. option:: -x
 
-    Use memstore as the objectstore backend for osds
+    Enable Cephx (on by default).
 
-.. option:: --cache <pool>
+.. option:: -X
 
-    Set a cache-tier for the specified pool
+    Disable Cephx.
```

Comment:
please use `CephX` or `cephx`, to be consistent with other parts of the document.

Reply:
ok, I will replace Cephx with cephx. Thanks.